src/pages/security/manage-access.md

```diff
@@ -80,6 +100,9 @@ In the example below, we'll use a 4.8 version of the OpenShift OAuth proxy.  See
             - name: oauth-proxy
               image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.8
               imagePullPolicy: IfNotPresent
+              env:
+              - name: GODEBUG
```

Comment:
Wonder if we want to call out a doc reference for this variable? Maybe this, https://pkg.go.dev/runtime#hdr-Environment_Variables, or was there some other reading that you found helpful? Consumers may question setting a debug variable without knowing/understanding what it is used for. I'll let you decide if it's necessary or not.